From 5abb8dd4f96f8ddc659e4fa653ee5ea386368c37 Mon Sep 17 00:00:00 2001
From: Alexander Martinz <amartinz@shiftphones.com>
Date: Wed, 30 Jan 2019 23:29:21 +0100
Subject: [PATCH 03/20] Eleven: remove guava dependency

Pulling in guava just for an annotation, which is available in the
support library as well is a bad idea.

Change-Id: I0e02943616c41077071e4ccba9125c73fb574d3b
Signed-off-by: Alexander Martinz <amartinz@shiftphones.com>
---
 Android.mk                                            | 3 +--
 src/org/lineageos/eleven/locale/LocaleSet.java        | 2 +-
 src/org/lineageos/eleven/locale/LocaleSetManager.java | 2 +-
 src/org/lineageos/eleven/locale/LocaleUtils.java      | 3 +--
 4 files changed, 4 insertions(+), 6 deletions(-)

diff --git a/Android.mk b/Android.mk
index 391067b..2a3ec5d 100644
--- a/Android.mk
+++ b/Android.mk
@@ -10,8 +10,7 @@ LOCAL_RESOURCE_DIR := $(addprefix $(LOCAL_PATH)/, res)
 
 LOCAL_STATIC_JAVA_LIBRARIES := \
     android-support-v8-renderscript \
-    android-common \
-    guava
+    android-common
 
 LOCAL_STATIC_ANDROID_LIBRARIES := \
     android-support-v4 \
diff --git a/src/org/lineageos/eleven/locale/LocaleSet.java b/src/org/lineageos/eleven/locale/LocaleSet.java
index 59949ac..0b0f3b0 100644
--- a/src/org/lineageos/eleven/locale/LocaleSet.java
+++ b/src/org/lineageos/eleven/locale/LocaleSet.java
@@ -16,8 +16,8 @@
 
 package org.lineageos.eleven.locale;
 
+import android.support.annotation.VisibleForTesting;
 import android.text.TextUtils;
-import com.google.common.annotations.VisibleForTesting;
 import java.util.Locale;
 
 public class LocaleSet {
diff --git a/src/org/lineageos/eleven/locale/LocaleSetManager.java b/src/org/lineageos/eleven/locale/LocaleSetManager.java
index de245f6..6e49f9e 100644
--- a/src/org/lineageos/eleven/locale/LocaleSetManager.java
+++ b/src/org/lineageos/eleven/locale/LocaleSetManager.java
@@ -17,11 +17,11 @@
 package org.lineageos.eleven.locale;
 
 import android.content.Context;
+import android.support.annotation.VisibleForTesting;
 import android.text.TextUtils;
 import android.util.Log;
 
 import org.lineageos.eleven.provider.PropertiesStore;
-import com.google.common.annotations.VisibleForTesting;
 
 import java.util.Locale;
 
diff --git a/src/org/lineageos/eleven/locale/LocaleUtils.java b/src/org/lineageos/eleven/locale/LocaleUtils.java
index 40af650..8f91639 100644
--- a/src/org/lineageos/eleven/locale/LocaleUtils.java
+++ b/src/org/lineageos/eleven/locale/LocaleUtils.java
@@ -18,13 +18,12 @@ package org.lineageos.eleven.locale;
 
 import android.provider.ContactsContract.FullNameStyle;
 import android.provider.ContactsContract.PhoneticNameStyle;
+import android.support.annotation.VisibleForTesting;
 import android.text.TextUtils;
 import android.util.Log;
 
 import org.lineageos.eleven.locale.HanziToPinyin.Token;
 
-import com.google.common.annotations.VisibleForTesting;
-
 import java.lang.Character.UnicodeBlock;
 import java.util.ArrayList;
 import java.util.Collections;
-- 
2.17.1

